src/main/resources/application.yml

```diff
@@ -100,6 +100,18 @@ features:
         update-prison-location: false
   apply-limited-access-markers: false
 
+interventions-delius:
+  nsi-mapping:
+    provider-code: CRS
+    staff-code: CRSUATU
+    team-code: CRSUAT
+    nsi-status: INPROG
+    service-category-to-nsi-type:
+      "[Accommodation]": CRS01
+      "[Education, Training and Employment (ETE)]": CRS02
+      "[Finance, Benefits and Debt (FBD)]": CRS03
+      "[Dependency and Recovery (D&R)]": CRS04
```

Comment:
🙋‍♂️ To note, the service category titles might change without a second thought; their UUIDs won't.

That said, I don't think this is worth changing now as we will have "contract type" mapping in the near future.

Reply:
+1 for mapping using the UUID -> Delius ref. We'll include this in the next ticket